### ISSUE TITLE
Contrast fixes

### ISSUE THREAD

**Issue by jechols** [uoregon-libraries/oregon-oni#36]
WAVE toolbar found a lot of contrast issues.  These should be trivial to fix with a touch of CSS.

**Comment by jechols** [comment]
I *think* this fixes all the "real" issues.  Some flags are still present, but don't seem to be properly considering background colors or something.
